static/app/views/issueList/draggableTabBar.tsx

```diff
@@ -25,7 +26,13 @@ export function DraggableTabBar(props: DraggableTabBarProps) {
     <Tabs>
       <DraggableTabList tabs={tabs} setTabs={setTabs} orientation="horizontal">
         {tabs.map(tab => (
-          <DraggableTabList.Item key={tab.key}>{tab.label}</DraggableTabList.Item>
+          <DraggableTabList.Item
+            key={tab.key}
+            count={tab.queryCount}
+            hasUnsavedChanges={tab.hasUnsavedChanges}
+          >
+            {tab.label}
+          </DraggableTabList.Item>
```

Comment:
I really do not like how the TabList is consuming the tabs in two ways - the `tabs` prop and the `<DraggableTabList.Item>` and I'm currently thinking about the best way to resolve this